ztools/x509/json_test.go

```diff
@@ -33,6 +33,9 @@ func (s *JSONSuite) SetUpTest(c *C) {
 	}
 
 	for _, test := range tests {
+		if test.Name()[0] == 46 {
```

Comment:
@dadrian 46 is ascii of "." , which tells it to skip hidden files. In particular .DS_Store that is literally in every folder on MacOS